app/views/users/edit.html.erb

```diff
@@ -72,18 +72,18 @@ minlength: User.password_length.min %>
   </div>
  <%= form_for @user, as: :user, url: users_path do |form| %>
```

Comment:
@7riumph even though on the front-end, you have a modal to prevent a user from submitting a form without any preferences checked, make sure the user can't just directly hit this POST route (i.e using curl) to bypass the front-end security

Reply:
I also set up a back-end db validation in de8e1c12325efaed10b40d54787f6674c07cd42e if that covers it. If not, where would I start with that?

Reply:
Ah I see. I forgot about that